tuneme/urls.py

```diff
@@ -21,6 +21,10 @@
 
     url(r'', include('molo.core.urls')),
     url(r'', include(wagtail_urls)),
+
+    url(r'', include('molo.profiles.urls',
+                namespace='molo.profiles'))
+                
```

Comment:
`wagtail.urls` should be the last entry in the urlpatters since that's a catchall so all other entries should be before this one, the order is important as it determines the pattern matching ordering.

Also I'd recommend we use `r'^profiles/'` as the prefix rather than `r''`.